sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -1627,10 +1627,8 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
   test("SPARK-33084: Add jar support Ivy URI in SQL") {
     val testData = TestHive.getHiveFile("data/files/sample.json").toURI
     withTable("t") {
-      // hive-catalog-core has some transitive dependencies which dont exist on maven central
```

Comment:
The "some transitive dependencies which dont exist on maven central" is "org.pentaho:pentaho-aggdesigner-algorithm", it's not a problem in Hive 2.3.10, see https://github.com/apache/hive/pull/4486